isso/js/app/i18n.js

```diff
@@ -3,7 +3,7 @@ define(["app/config", "app/i18n/bg", "app/i18n/cs", "app/i18n/da",
         "app/i18n/fr", "app/i18n/hr", "app/i18n/ru", "app/i18n/it",
         "app/i18n/eo", "app/i18n/sv", "app/i18n/nl", "app/i18n/el_GR",
         "app/i18n/es", "app/i18n/vi", "app/i18n/zh_CN", "app/i18n/zh_CN", "app/i18n/zh_TW", "app/i18n/hu"],
-        function(config, bg, cs, da, de, en, fa, fi, fr, hr, ru, it, eo, sv, nl, el, es, vi, zh, zh_CN, zh_TW) {
+        function(config, bg, cs, da, de, en, fa, fi, fr, hr, hu, ru, it, eo, sv, nl, el, es, vi, zh, zh_CN, zh_TW) {
```

Comment:
Wait, did I do this wrong? "app/il8n/hu" is the last element in the prior array. I'm wondering if this will screw up translations for anything after `hr`? Seems like the best things is to put `app/i18n/hu` into alphabetical order (as all the others are)?

Reply:
Yup it's definitely broken. Will fix this.

![image](https://user-images.githubusercontent.com/4132322/33107067-75037d62-cf03-11e7-9379-d78906b3acbd.png)